general_rules/PenaltiesBonuses.tex

```diff
@@ -42,6 +42,26 @@ \subsection{Bonus for outstanding performance}\label{rule:outstanding_performanc
 	\item It is the decision of the \iaterm{Technical Committee}{TC} if (and to which degree) the bonus score is granted.
 \end{enumerate}
 
+\subsection{Bonus for perceived social intelligence}\label{rule:perceived_intelligence}
+\begin{enumerate}	
+    \item For the test \iterm{Receptionist} in \iterm{Stage~I} and, \iterm{Restaurant} in \iterm{Stage~II} tests. Teams are allowed to request an assessment of the robot's performance regarding its perceived social intelligence.
+    
+    \item This bonus, ranging from 0 to 50, depends on the robot's social performance which will be assessed by Referees using a specially designed scale in a survey.
```

Comment:
I think we should name the categories that are asked (although this makes the rulebook longer). Not only referees, but also others who only fill out this survey for a task should be allowed. Also, I don't think we've designed a specific scale, but maybe I'm just misunderstanding. The survey should also be linked here. 

1.
This bonus, which ranges from 0 to 50, depends on the robot's social performance, which is assessed by a group of evaluators (constant for the task) using a \href{https://forms.office.com/Pages/ResponsePage.aspx?id=6sSEXw03nkuDDHVvi_G1H7VNGCdGFtZJs0ryJVVWtCFUQVFSWDlYM0FHRVA2QllIT0tOQjI2QUcxQi4u}{survey}. The following categories are recorded on a linear scale of 1-5 and all values are averaged: Perception, social interaction, recognition, observation, adaptation, prediction, support, trustworthiness, empathy, harm avoidance, social awareness, arrogance, learning, manners, social competence.

2.
This bonus, which ranges from 0 to 50, depends on the robot's social performance, which is assessed by a group of evaluators (constant for the task) using a \href{https://forms.office.com/Pages/ResponsePage.aspx?id=6sSEXw03nkuDDHVvi_G1H7VNGCdGFtZJs0ryJVVWtCFUQVFSWDlYM0FHRVA2QllIT0tOQjI2QUcxQi4u}{survey}. The rated categories are recorded on a linear scale from 1-5 and all values are then averaged.

Reply:
Yes please add a scoring item to the tests' scoresheets.
I think it's fine if the form is linked once...